connector/loki/loki.go

```diff
@@ -252,7 +275,18 @@ func (client *LokiConnector) send() (*http.Response, error) {
 		return nil, err
 	}
 
-	response, err := http.Post(client.url+client.endpoints.push, "application/json", bytes.NewReader(str))
+	request, err := http.NewRequest("POST", client.url+client.endpoints.push, bytes.NewReader(str))
+	if err != nil {
+		client.logger.Metadata(map[string]interface{}{
+			"error": err,
+		})
+		client.logger.Error("An error occured when trying to construct a request to send logs")
+		return nil, err
+	}
+	request.Header.Add("X-Scope-OrgID", client.tenantID)
+	request.Header.Add("Content-Type", "application/json")
+	var c http.Client
```

Comment:
This looks like an uninitialized Client struct. I don't want to be paranoild, but shouldn't there be rather: c := &http.Client{} ?

Reply:
I'm not a go expert, but isn't everything in go initialized to its "zero" value, which in this case I think would be http.Client{}? This worked when I tested it, but I'll change it to what you suggested just to be sure.